sidechain/state/src/impls.rs

```diff
@@ -39,10 +39,56 @@ where
 		self.ext.hash()
 	}
 
+	fn ext(&self) -> &Self::Externalities {
+		&self.ext
+	}
+
+	fn ext_mut(&mut self) -> &mut Self::Externalities {
+		&mut self.ext
+	}
```

Comment:
In order to have a clean interface, we should not expose inner members like this. But I suppose that's the way it has been before and changing it would not fit the scope of this PR?

Reply:
Yes, unfortunately we must expose them currently, so we can pass the `Externalities` to the `STF`.